data/gui/window/game_version.cfg

```diff
@@ -624,6 +624,18 @@
 								[/button]
 							[/column]
 
+							[column]
+								horizontal_alignment = left
+								border = all
+								border_size = 5
+
+								[button]
+									id = "run_migrator"
+									#po: button to open the version migration dialog manually to re-migrate add-ons and the preferences file
+									label = _ "Migrate"
```

Comment:
This might need a bit more text or a tooltip?

Reply:
I was planning to backport this, so I was thinking I'd add the tooltip for 1.19 after merging and backporting this PR.